resources/src/main/java/org/robolectric/manifest/AndroidManifest.java

```diff
@@ -831,7 +832,8 @@ public Path getApkFile() {
 
   /** @deprecated Do not use. */
   @Deprecated
-  public boolean supportsLegacyResourcesMode() {
+  @InlineMe(replacement = "true")
+  public final boolean supportsLegacyResourcesMode() {
```

Comment:
Maybe this method needs to mark with `@SuppressWarnings("InlineMeSuggester")` instead `@InlieMe`. This method is deprecated without replacement. And we maybe remote it someday.

Reply:
um... if we have to remove it someday, then until that time having it as inline make sense, isn't it? as a memory enhancement

Reply:
I don' think the `true` is more meaningful than a method(`supportsLegacyResourcesMode`) calling.

Reply:
ahh okay 👍🏻